docs-client/src/containers/HomePage/index.tsx

```diff
@@ -14,11 +14,58 @@
  * under the License.
  */
 
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Tooltip from '@material-ui/core/Tooltip';
 import Typography from '@material-ui/core/Typography';
 import React from 'react';
+import Section from '../../components/Section';
+import {
+  convertLongToUTCDate,
+  formatReadableTime,
+  Versions,
+} from '../../lib/versions';
 
-export default class HomePage extends React.PureComponent {
+interface OwnProps {
+  versions?: Versions;
+}
+
+type Props = OwnProps;
+
+export default class HomePage extends React.PureComponent<Props> {
   public render() {
+    const { versions } = this.props;
+    const data = versions;
+    const tableRows: any = [];
+
+    if (data) {
+      data.getVersions().forEach((version) => {
+        tableRows.push(
+          <TableRow>
+            <TableCell>{version.artifactId}</TableCell>
+            <TableCell>
+              {version.artifactVersion}
+              {data.getArmeriaArtifactVersion() !== version.artifactVersion
+                ? '(!)'
+                : ''}
+            </TableCell>
+            <TableCell>
+              <Tooltip title={convertLongToUTCDate(version.commitTimeMillis)}>
+                <Typography variant="body2">
+                  {formatReadableTime(version.commitTimeMillis)}
+                </Typography>
+              </Tooltip>
+            </TableCell>
+            <TableCell>{version.shortCommitHash}</TableCell>
```

Comment:
Could wrap the short commit hash with `<Tooltip title={version.longCommitHash}/>` ?

Reply:
It's a good idea~